subprojects/build-events/src/main/java/org/gradle/internal/build/event/types/DefaultBuildBuildDescriptor.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.build.event.types;
+
+import org.gradle.api.NonNullApi;
+import org.gradle.internal.operations.OperationIdentifier;
+
+@NonNullApi
+public class DefaultBuildBuildDescriptor extends DefaultOperationDescriptor {
```

Comment:
```suggestion
public class DefaultRunBuildDescriptor extends DefaultOperationDescriptor {
```

🤔 Not sure if that name is better. `BuildBuild` is strange for sure. Should there also be an interface?

Reply:
Now it's `DefaultRootOperationDescriptor`